nobo.py

```diff
@@ -482,6 +484,10 @@ def response_handler(self, r):
     def create_override(self, mode, type, target_type, target_id='-1', end_time='-1', start_time='-1'):
         command = [self.API.ADD_OVERRIDE, '1', mode, type, end_time, start_time, target_type, target_id]
         self.send_command(command)
+        for o in self.overrides:
+            if self.overrides[o]['target_id'] == target_id:
+                self.overrides[o]['mode'] = mode
+                self.overrides[o]['type'] = type
```

Comment:
I am not sure about this change, usually we wait for the EcoHub to broadcast the changes, and then `self.overrides` get updated at https://github.com/echoromeo/pynobo/blob/e6e578647971ccaa263bdeef9fa29b09603e021a/nobo.py#L434

I would prefer waiting for the update from the EcoHub, before changing the structure (similar for lines 504 and 507). What are the reasons we cannot wait for the update here?

Reply:
The reason is to get a better user experience in Home Assistant. As it was without this modification a change by the user would seem like it was rejected because it would go back to the previous setting and then back to the new setting some time later. It was confusing for users not knowing what was going on.

And if the temperature is set to a new value it is likely that this new value is the correct one in the system and not the old one. I dont see any good reason why it should go back to the old value for a few seconds.

Reply:
OK, that makes sense. I don't really have a preference either way.